integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -61,19 +61,9 @@ const testRunWorkflowOnSnapshotFn = _.flow(
   await delay(1000) // Without this delay, the input field sometimes reverts back to its default value
   await click(page, clickable({ text: 'Save' }))
 
-  await delay(1000) // The Run Analysis button requires time to become enabled after hitting the save button
```

Comment:
FYI @ungwudik, I am sharing this increase in timeout and logging between the 2 run-workflow tests. This is certainly not the only cause of run-workflow-on-snapshot failing (https://broadworkbench.atlassian.net/browse/QA-1700), but it might help reduce the number of failures.

Reply:
Thanks @cahrens!
I have not started on QA-1700, but will keep your updates here (and in the ticket itself) in mind.